apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json

```diff
@@ -4,17 +4,37 @@
   "info": {
     "singularName": "homepage",
     "pluralName": "homepages",
-    "displayName": "Homepage"
+    "displayName": "Homepage",
+    "description": ""
   },
   "options": {
     "draftAndPublish": true
   },
-  "pluginOptions": {},
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
```

Comment:
why there is this change?

Reply:
I was very hesitant, thanks!

Reply:
As the localization of the homepage was missing, the response to the GET `/api/homepage?populate%5BcomingsoonDocumentation%5D%5Bpopulate%5D%5B0%5D=links&populate%5BproductsShowcase%5D%5Bpopulate%5D%5B0%5D=products.logo` was not showing the localized `productsShowcase` and `comingsoonDocumentation` sections